app/docker/networks/network.service.ts

```diff
@@ -0,0 +1,39 @@
+import axios, { parseAxiosError } from '@/portainer/services/axios';
+import { EnvironmentId } from '@/portainer/environments/types';
+
+import { NetworkId } from './types';
+
+const systemNetworks: string[] = ['host', 'bridge', 'none'];
+
+export function isSystemNetwork(networkName: string): boolean {
```

Comment:
this should be moved to `network.helper.ts`

Reply:
updated